CONTRIBUTING.md

```diff
@@ -1,98 +1,211 @@
 # Contributing
 
-Thank you for considering contributing to tfsec! 
+Thank you for considering contributing to tfsec! :heart:
 
-We've documented the process of adding a new check below. If you have any other specific questions/problems that are preventing you from raising a PR, please get in touch with us! You can [find us on Slack](https://join.slack.com/t/tfsec/shared_invite/zt-o6c7mgoj-eJ1sLDv595sKiP5OPoHJww) - or simply [raise an issue](https://github.com/aquasecurity/tfsec/issues/new) and we'll do our best to help you out.
+<p align="center">
+    <img alt="welcome to the party, pal!" src="https://media.giphy.com/media/l0MYGb1LuZ3n7dRnO/giphy.gif"/>
+</p>
 
-## Adding a New Check
+Please consider checking out the following resources:
 
-Adding a check typically involves the addition of two files. The first is the file containing the check code itself and it's documentation. The second is a file containing tests for your check. You won't typically need to touch any other files - documentation is generated automatically from the check file itself.
+- [ARCHITECTURE.md](ARCHITECTURE.md): A very high level document that gives an overview of the code and aims to answer the question *Where is the code that does X?*
+- [#tfsec on AquaSec Slack](https://slack.aquasec.com): Come and talk over any questions/suggestions you have with us on Slack!
+- [tfsec documentation](https://aquasecurity.github.io/tfsec/latest/): General usage documentation and rule information.
 
-Adding a check can be simplified by running the `make new-check` command. this will request base information about the new check then generate the skeleton code for you to populate.
+## :ballot_box_with_check: PR Checklist
 
-Key attributes requested;
+- Ensure the build passes locally first with `make pr-ready`.
+- Add a brief description of your change.
+- Link to the issue which the PR resolves. Please [create one](https://github.com/aquasecurity/tfsec/issues/new/choose) if necessary.
+- Prefix your PR title with one of [these](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) as described in [Conventional Commits](https://www.conventionalcommits.org/) e.g. `feat: Add new rule for AWS S3 bucket encryption`.
```

Comment:
I think we should add the list here so you don't need to link out - but still include the link